AUTHORS

```diff
@@ -245,27 +264,29 @@ Timo Reimann                            ttr314@googlemail.com
 Tobias Klauser                          tobias@cilium.io
 Tobias Kohlbau                          tobias@kohlbau.de
 Tom Hadlaw                              thomas.hadlaw@hootsuite.com
+Tomoki Sugiura                          cheztomo513@gmail.com
 Tom Payne                               tom@isovalent.com
 Tony Lambiris                           tony@criticalstack.com
 Tony Lu                                 tonylu@linux.alibaba.com
 Travis Glenn Hansen                     travisghansen@yahoo.com
 Trevor Roberts Jr                       Trevor.Roberts.Jr@gmail.com
 trevor tao                              trevor.tao@arm.com
 Umesh Keerthy B S                       umesh.freelance@gmail.com
-unixdaddy                               unixdaddy@gmail.com
 Vadim Ponomarev                         velizarx@gmail.com
 Valas Valancius                         valas@google.com
+Vance Li                                liyannois@gmail.com
 Vigneshwaren Sunder                     vickymailed@gmail.com
 Vishnu Soman K                          vishnusomank05@gmail.com
 Vlad Artamonov                          742047+vladdy@users.noreply.github.com
 Vlad Gorodetsky                         v@gor.io
 Vladimir Nachev                         vladimir.penkov.nachev@gmail.com
-Vlad Ungureanu                          vladu@palantir.com
+Vlad Ungureanu                          ungureanuvladvictor@gmail.com
```

Comment:
Why did this change? I didn't see any updates to Vlad's email in the `.mailmap` commit

Reply:
Hmm, well @ungureanuvladvictor  is not in the .mailmap at all so that might be the reason. I'll add him into the mailmap with his Palantir address and fix this up. Good catch.

Reply:
FWIW, I checked that all other changes to the authors lined up with changes in `.mailmap` commit, except for this one